src/main/scala/org/mdedetrich/stripe/v1/Plans.scala

```diff
@@ -124,95 +128,149 @@ object Plans extends LazyLogging {
        x.intervalCount,
        x.livemode,
        x.metadata,
-       x.name,
+       x.nickname,
+       x.product,
        x.statementDescriptor,
        x.trialPeriodDays))
 
+  sealed abstract class Product
```

Comment:
From [the documentation](https://stripe.com/docs/api#create_plan):

> product: The product whose pricing the created plan will represent. This can either be the _ID of an existing product_, or a dictionary containing _fields used to create a service product_.

The names for the two sub classes, `ProductId` and `ServiceProduct` from the names of the two italicized phrases, but I'm happy to shorten them if that would be more consistent with the rest of the names.